Scrapper-Service/database/mdb.py

```diff
@@ -48,23 +49,26 @@ def __init__(   self  , logger:Logger ,  database_name:str , collection_name:str
                 collection.create_index([(idx , pymongo.ASCENDING )] , unique = unique )
                 
         except Exception as e:
-            self.logger.error("Failed to initiate mongodb client error: {}".format(e))
+            self.logger.error("Failed to initiate mongodb client error: {}".format(e),)
```

Comment:
* It may be a dumb suggestion but for ur recent comment of issue u can try and remove that ',' at the very end. Definitely it should not be the case. But it's unable to find the msg argument. which earlier was not the case